tools/SetupFlow/DevHome.SetupFlow/ViewModels/PackageViewModel.cs

```diff
@@ -203,22 +202,19 @@ private void ToggleSelection()
         _screenReaderService.Announce(announcementText);
     }
 
-    /// <summary>
-    /// Gets the package icon based on the provided theme
-    /// </summary>
-    /// <param name="theme">Package icon theme</param>
-    /// <returns>Package icon</returns>
-    private BitmapImage GetIconByTheme(RestoreApplicationIconTheme theme)
+    public BitmapImage IconByTheme
     {
-        return theme switch
+        get
         {
-            // Get default dark theme icon if corresponding package icon was not found
-            RestoreApplicationIconTheme.Dark =>
-                _package.DarkThemeIcon == null ? DefaultDarkPackageIconSource : CreateBitmapImage(_package.DarkThemeIcon),
-
-            // Get default light theme icon if corresponding package icon was not found
-            _ => _package.LightThemeIcon == null ? DefaultLightPackageIconSource : CreateBitmapImage(_package.LightThemeIcon),
-        };
+            if (_themeSelector.IsDarkTheme())
+            {
+                return CreateBitmapImage(_package.DarkThemeIcon);

```

Comment:
Why are we now guaranteed _package.DarkThemeIcon is never null?

Reply:
We're not. Thank you for catching this! I missed the null check in the switch statement I cut out. It looks like I only need to swap RestoreApplicationIconTheme with theme. What do you think?